app/src/main/java/com/framgia/fsalon/screen/imagecustomer/ImageCustomerViewModel.java

```diff
@@ -0,0 +1,93 @@
+package com.framgia.fsalon.screen.imagecustomer;
+
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.support.v4.app.Fragment;
+import android.view.View;
+
+import com.android.databinding.library.baseAdapters.BR;
+import com.codewaves.stickyheadergrid.StickyHeaderGridLayoutManager;
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.data.model.ListImageResponse;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.data.model.ListImageResponse. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports